### PR TITLE
WordPress doc

### PR DESCRIPTION
**Pull request by mattmanning**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;RMW4If8T&#x2F;321-doc-for-wordpress-using-shared-volumes"> doc for wordpress using shared volumes</a>

File: _languages/wordpress.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+title: "WordPress"
+---
+
+[WordPress](https://wordpress.org/) is a popular blogging platform based on PHP and MySQL. It's simple to set up a WordPress blog on Convox with the following steps.
+
+## Create the App and Database
+
+```
+$ convox apps create wordpress
+$ convox services create mysql --name wordpress-db
```
Comment (ianmalott) [inline _languages/wordpress.md]: Wondering if someone new to Convox might find `wordpress` on line 10 confusing. Using something like `examplewordpressapp` or `wordpressexample` might make it clearer that the app does not need to be named "wordpress" to work, unlike the similar-looking `create mysql` command on line 11, in which the type `mysql` is more significant.

File: _languages/wordpress.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  ports:
+    - 80:80
+    - 443:80
+  volumes:
+    - /var/www/html
+```
+
+The [official WordPress image from Docker Hub](https://hub.docker.com/_/wordpress/) will be used in your app.
+
+The `volumes` directive will persist all of your WordPress files to a network filesystem associated with your Rack. This means your themes, plugins, and uploaded media will not be lost on restarts, and can be shared between containers, supporting scaling.
```
Comment (ianmalott) [inline _languages/wordpress.md]: Could link `network filesystem` to the volumes doc or add a parenthetical mentioning EFS.

File: _languages/wordpress.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Status  running
+Exports
+  URL: mysql://app:EXWKHJKDZTBQIPGUMEKGSNTRYGAYAC@convox-dev-wordpress-db.cbm4183bzjrr.us-east-1.rds.amazonaws.com:3306/app
+```
+
+Set the RDS database environment variables on the app:
+
+```
+$ convox env set WORDPRESS_DB_HOST=convox-dev-wordpress-db.cbm4183bzjrr.us-east-1.rds.amazonaws.com:3306 WORDPRESS_DB_USER=app WORDPRESS_DB_PASSWORD=EXWKHJKDZTBQIPGUMEKGSNTRYGAYAC WORDPRESS_DB_NAME=app
+Updating environment... OK
```
Comment (ianmalott) [inline _languages/wordpress.md]: This line's pretty long and horizontal scrolling isn't fun. Could we break it up with some continuations `\`?

File: _languages/wordpress.md (hunk 1 above)
Comment (mwarkentin) [inline _languages/wordpress.md]: 👍 `my-wordpress-site` or similar might be good too

File: _languages/wordpress.md (hunk 3 above)
Comment (mwarkentin) [inline _languages/wordpress.md]: You could also break it up into a few separate `convox env set`s

**Pull request merged**